Homework-Functions/src/sciit/homework/functions/ex2/ConvertionFahrenheitToCelsius.java

```diff
@@ -0,0 +1,14 @@
+package sciit.homework.functions.ex2;
+
+public class ConvertionFahrenheitToCelsius {
+    public static void main(String[] args) {
+
+    }
+
+
+    public static double convertFahrenheitToCelsius(double valueF) {
```

Comment:
Desi functioneaza si asa, metoda `convertFahrenheitToCelsius` ar fi trebuit scrisa in clasa `BodyTemperature`, nu vad nici un motiv pentru care functia asta sa merite propriul fisier.